src/main/java/seedu/address/model/pdf/Deadline.java

```diff
@@ -112,7 +116,7 @@ public boolean isMet() {
      * @return - existence of localdate.
      */
     public boolean exists() {
-        return !(this.date.equals(LocalDate.MIN));
+        return !(this.date.equals(LocalDate.MIN)) || this.status == STATUS_REMOVE;
```

Comment:
If we are using an `Enum` then there will be no need to use the `LocalDate.MIN` anymore. Hmm i think that'd mean that you gotta revamp whole parts of the way we treat the Deadline Class.